patches/0004-De-deprecate-bucket_object.patch

```diff
@@ -5,13 +5,13 @@ Subject: [PATCH] De-deprecate bucket_object
 
 
 diff --git a/internal/service/s3/bucket_object.go b/internal/service/s3/bucket_object.go
-index 59702972dc..03c4dec17c 100644
+index 36c4378a5a..18307157af 100644
 --- a/internal/service/s3/bucket_object.go
 +++ b/internal/service/s3/bucket_object.go
 @@ -68,7 +68,7 @@ func resourceBucketObject() *schema.Resource {
  				Computed: true,
  			},
- 			"bucket": {
+ 			names.AttrBucket: {
 -				Deprecated:   "Use the aws_s3_object resource instead",
 +				// FORK: Stack72 removed the deprecation warning as it was misleading for Pulumi users
```

Comment:
This is a good find, relevant to some S3 revisiting work that I'm doing, going to log that.